### PR TITLE
Reyna's card

### PR DESCRIPTION
**Pull request by reynalau220**
added a new card

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,6 +121,31 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Reyna Lau</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/your_user_handle" target="_blank">reynalau220</a>
```
Comment (StackOverflowIsBetterThanAnyAI) [inline index.html:129]:
```suggestion
            <a href="https://github.com/reynalau220" target="_blank">reynalau220</a>
```

**Review by StackOverflowIsBetterThanAnyAI — COMMENTED** [review]
Please update the link of your GitHub profile.

**Review by StackOverflowIsBetterThanAnyAI — COMMENTED** [review]
looks good now

**Comment by theAravinthM** [comment]
Congratulations @reynalau220 on your PR Merge, and Thanks for your Contribution! Your card will be live on our Website in a moment!

Now you can follow the Next Steps. You are also welcome to join our Discord Server: https://discord.gg/tWkvS4ueVF and be a part of our community.

Happy contributing!

**Pull request merged**